examples/USER/misc/grem/lj-temper/0/log.lammps.0

```diff
@@ -29,133 +25,1044 @@ fix             fxgREM all grem ${lambda} -.03 -30000 fxnpt
 fix             fxgREM all grem 900 -.03 -30000 fxnpt
 thermo_modify   press fxgREM_press
 
-temper/grem     10000 100 ${lambda} fxgREM fxnpt 10294 98392 #${walker}
-temper/grem     10000 100 900 fxgREM fxnpt 10294 98392 
+thermo          10
+thermo_style    custom step temp f_fxgREM pe etotal press vol
+WARNING: New thermo_style command, previous thermo_modify settings will be lost (../output.cpp:705)
```

Comment:
Nope, thanks for the catch. This is specific to the example file though, I'll update that.